src/FillArrays.jl

```diff
@@ -633,13 +655,12 @@ if VERSION ≥ v"1.5"
         print(io, Base.dims2string(length.(inds)), " Eye{$T}")
 end
 
-Base.show(io::IO, ::MIME"text/plain", x::Union{Eye,AbstractFill}) = show(io, x)
 
 ##
 # interface
 ##
 
-getindex_value(a::LinearAlgebra.AdjOrTrans) = getindex_value(parent(a))
+getindex_value(a::LinearAlgebra.AdjOrTrans) = getindex_value(parent(a)) # FillArrays.getindex_value(Adjoint(Fill(1+im,2,2)))
```

Comment:
I didn't mean to check this in, but think this is an unrelated bug -- adjoint is recursive.

Reply:
You are right. Can you fix this too and add a test?

Reply:
Sure, done!